src/utils/HSwitchGlobalVars.res

```diff
@@ -1,5 +1,8 @@
 @val external appVersion: string = "appVersion"
 
+let hyperSwitchAppVersion =
+  Window.env.versionNumber->Option.getOr(DateTimeUtils.getCurrentUTCTime(~separator=".", ()))
+
```

Comment:
Do not show the current time if env is not present as the latest version and current time could be different

Reply:
It shouldn't matter. Why it will not be there? If not there then it's our fault I guess in deploying.

Reply:
Reasons why we shouldn't take current time as default value : 
1. Its giving the time in UTC
2. Sometimes the tag creation is done manually . So the latest version could differ 
3. Changes in tag format will affect this . 
4. And ideally the app version should come from github tag list itself .

Reply:
It's not about tag it should be done by us. It's okay. No issues. Just remove it @gitanjli525